packages/web/docs/src/pages/docs/schema-registry/external-schema-composition.mdx

```diff
@@ -1,9 +1,4 @@
-import NextImage from 'next/image'
 import { Callout, Steps, Tabs } from '@theguild/components'
-import externalCompositionInvalid from '../../../../public/docs/pages/management/external-composition-invalid.png'
```

Comment:
Just thinking — would it be possible to colocate the images inside of `docs` (content) and the use relative imports instead of absolute paths to public?